File: Python/baekjoon/단계별로 풀어보기/기본 수학2/11653_소인수분해.py
# https://www.acmicpc.net/problem/11653

n = int(input())
cond = True
for i in range(2, int(n ** 0.5) + 1):
    while not n % i:
        print(i)
        n //= i
        cond = False
if n > 1:
    print(n)

File: C#/Baekjoon/약수, 배수와 소수/11653_소인수분해.cs
//https://www.acmicpc.net/problem/11653

int n = int.Parse(Console.ReadLine());
if (n == 1) return;
for (int i = 2;i <= n; i++)
{
    while (n % i == 0)
    {
        Console.WriteLine(i);
        n /= i;
    }
}